CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Added
 - **Dictionary**
   - Added `Dictionary[path:]` subscript for deep fetching/setting nested values. [#574](https://github.com/SwifterSwift/SwifterSwift/pull/573) by [@calebkleveter](https://github.com/calebkleveter)
+- **UIActivity**
+  - Added `ActivityType` constants for iCloud Drive, WhatsApp, LinkedIn, XING and Google Plus. [#580](https://github.com/SwifterSwift/SwifterSwift/pull/580) by [staffler-xyz](https://github.com/staffler-xyz)
```

Comment:
I don't see the Google+ addition. Also, isn't it being discontinued or something?